modules/test-structure/validate_struct.go

```diff
@@ -11,11 +11,24 @@ import (
 	"github.com/mattn/go-zglob"
 )
 
+// ValidateFileType is the underlying module type to search for when performing validation. Either Terraform or Terragrunt
+// files are targeted during a given validation sweep
+type ValidateFileType string
+
+const (
+	// TF represents repositories that contain Terraform code
+	TF = "*.tf"
+	// TG represents repositories that contain Terragrunt code
+	TG = "*.hcl"
```

Comment:
This should search for `terragrunt.hcl`, not any `hcl` file. Otherwise, it will catch packer templates too (which end in `pkr.hcl`).

Reply:
Thanks - I wasn't sure about this - and initially had it searching for `terragrunt.hcl`. I will switch it back.

Reply:
Done in 2b755e7